### PR TITLE
Update golangci-lint (and indirect deps)

### PR DESCRIPTION
**Pull request by 2uasimojo**
This is mainly to get an updated version of an indirect dependency,
github.com/denis-tingajkin/go-header, to get rid of some suspicious
unicode characters in its README to avoid getting flagged for
CVE-2021-42574, CVE-2021-42694. See
https://access.redhat.com/security/vulnerabilities/RHSB-2021-007 for
more on those.

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -438,8 +470,9 @@ github.com/davegardnerisme/deephash v0.0.0-20210406090112-6d072427d830 h1:gn7TsP
 github.com/davegardnerisme/deephash v0.0.0-20210406090112-6d072427d830/go.mod h1:ToGe2SdaElKXzEmYLttAgFHy0exxh0wyq9zG7ZjjjYM=
 github.com/daviddengcn/go-colortext v0.0.0-20160507010035-511bcaf42ccd/go.mod h1:dv4zxwHi5C/8AeI+4gX4dCWOIvNi7I6JCSX0HvlKPgE=
 github.com/dchest/bcrypt_pbkdf v0.0.0-20150205184540-83f37f9c154a/go.mod h1:Bw9BbhOJVNR+t0jCqx2GC6zv0TGBsShs56Y3gfSCvl0=
-github.com/denis-tingajkin/go-header v0.3.1 h1:ymEpSiFjeItCy1FOP+x0M2KdCELdEAHUsNa8F+hHc6w=
 github.com/denis-tingajkin/go-header v0.3.1/go.mod h1:sq/2IxMhaZX+RRcgHfCRx/m0M5na0fBt4/CRe7Lrji0=
+github.com/denis-tingajkin/go-header v0.4.2 h1:jEeSF4sdv8/3cT/WY8AgDHUoItNSoEZ7qg9dX7pc218=
+github.com/denis-tingajkin/go-header v0.4.2/go.mod h1:eLRHAVXzE5atsKAnNRDB90WHCFFnBUn4RN0nRcs1LJA=
```
Comment (2uasimojo) [inline go.sum:475]: This.

Comment (joelddiaz) [reply go.sum:475]: what were the 'go mod' 'go get' commands run to get this PRs changeset?

**Review by joelddiaz — COMMENTED** [review]
/lgtm

**Pull request merged**